### ISSUE TITLE
Building problem on Linux

### ISSUE THREAD

**Issue by ATM-Jahid** [njoy/NJOY21#96]
After executing `make` in the `NJOY21/bin` directory, the building process gets stuck at 84%. My system crashes at this point. The last thing that the shell shows is the following.

`[ 84%] Building CXX object CMakeFiles/njoy21.dir/src/main.cpp.o`

**Comment by jlconlin** [comment]
When you say it gets "stuck", what do you mean? How long is it stuck? Does it make the `njoy21` binary?

**Comment by ATM-Jahid** [comment]
By stuck, I mean my system becomes unresponsive totally. 
```
% sudo make
[ 21%] Built target utility 
[ 78%] Built target njoy 
[ 82%] Built target njoy_c_bindings 
[ 84%] Building CXX object CMakeFiles/njoy21.dir/src/main.cpp.o
```
After that I have to restart it. 

Right after executing `make` command, everything seems normal but after reaching that 84%, CPU usage  skyrockets (I am assuming ram usage does too) and at some point my laptop becomes unresponsive to anything. The result is the same when I run more than one job (`make -j 4`)at a time. 

What is the destination of the `njoy21` binary? I have checked `/usr/bin/` and `/usr/sbin/`. There is no njoy21 file there.

**Comment by jlconlin** [comment]
@ATM-Jahid If you have a slow machine, or if there isn't much memory, this could cause some problems. NJOY21 does require a fair amount of memory to compile. It would be easier on the machine if you just did one job (`make -j1`).

**Comment by ATM-Jahid** [comment]
Well, I am not sure if memory is the problem. I ran `make` and `make test` on the same machine [before](https://github.com/njoy/NJOY21/issues/91#issuecomment-512552076).

**Comment by jlconlin** [comment]
`make test` likely uses less memory than `make` does.

**Comment by ATM-Jahid** [comment]
I remember running `make` that time. It went smoothly. I was running other programs in parallel and I didn't face this sort of problem then. 

And, exactly how memory intensive is the process?

**Comment by jlconlin** [comment]
I don't know exactly off the top of my head. I wouldn't be surprised if it consumed up to 4 GB.

**Comment by marton384** [comment]
Hey guys! I managed to compile/install the program after solving several issues with permissions and config commands. Yet when I try to run an input I get the following error message:

 reconr...                                                                0.0s
At line 212 of file /home/base/NJOY21/subprojects/njoy/src/util.f90 (unit = 20)
Fortran runtime error: Cannot open file 'tape20': No such file or directory

This is not clear so far: should I provide this 'tape20' file or the program was supposed to find it somewhere? When I run the 'make test' command everything is OK, but I cannot run any other input because the code cannot open the tape the input refers to. I checked the NJOY2016 manual but there is no information on such issues. Thanks!!!
